tests/test_factory.py

```diff
@@ -35,7 +35,7 @@
     "conf, ds_config, service_type",
     [
         (AwardsVocabularyConfig(), awards_ds_config, "awards-service"),
-        (NamesVocabularyConfig(), names_ds_config, "names-service"),
+        (NamesVocabularyConfig(), names_ds_config, "async"),
```

Comment:
what are testing here for? not sure I understand what's happening

Reply:
It's checking for the expected writer, now changed to the async writer